### ISSUE TITLE
Missing panel css'es from static/extensions/panel/css in 0.11.1 compared to 0.10.3

### ISSUE THREAD

**Issue by JackDapid** [holoviz/panel#2156]
#### ALL software version info
bokeh 2.2.3 & panel 0.10.3  vs bokeh 2.3.0 & panel 0.11.1

#### Description of expected behavior and the observed behavior
in bokeh 2.2.3 & panel 0.10.3 necessary css's are loaded with ReactTemplate, so html shows:
```
<link rel="stylesheet" href="static/extensions/panel/css/alerts.css" type="text/css" />
<link rel="stylesheet" href="static/extensions/panel/css/card.css" type="text/css" />
<link rel="stylesheet" href="static/extensions/panel/css/dataframe.css" type="text/css" />
<link rel="stylesheet" href="static/extensions/panel/css/json.css" type="text/css" />
<link rel="stylesheet" href="static/extensions/panel/css/markdown.css" type="text/css" />
<link rel="stylesheet" href="static/extensions/panel/css/widgets.css" type="text/css" />
```
while these are missing using bokeh 2.3.0 & panel 0.11.1 - seems to be related to changes in https://github.com/holoviz/panel/blob/87100776b0f369e3a19bf45d26a18e152a0c3b91/panel/template/base.py#L464
but will start to look into it now

#### Complete, minimal, self-contained example code that reproduces the issue
A min example taken from another issue from @MarcSkovMadsen 
```
import hvplot.pandas  # pylint: disable=unused-import
import pandas as pd
import panel as pn

COLOR = "#E1477E"

def _get_chart_data() -> pd.DataFrame:
    """## Chart Data

    Returns:
        pd.DataFrame -- A DataFrame with dummy data and columns=["Day", "Orders"]
    """

    chart_data = {
        "Day": [
            "Sunday",
            "Monday",
            "Tuesday",
            "Wednesday",
            "Thursday",
            "Friday",
            "Saturday",
        ],
        "Orders": [
            15539,
            21345,
            18483,
            24003,
            23489,
            24092,
            12034,
        ],
    }
    return pd.DataFrame(chart_data)

def _holoviews_chart():
    """## Dashboard Orders Chart generated by HoloViews"""
    data = _get_chart_data()
    line_plot = data.hvplot.line(
        x="Day",
        y="Orders",
    )
    fig = line_plot
    return fig

chart = _holoviews_chart()
app = pn.pane.HoloViews(chart)
btn = pn.widgets.Button(name="Feedback", button_type="primary")

react = pn.template.ReactTemplate(
    title="Min React Test",
    header_background=COLOR,
    header_color="#ffffff",
)

react.sidebar.append(btn)
react.main[:4, :6] = pn.Card(app, title='Card')

react.servable()
```

#### Screenshots or screencasts of the bug in action
loaded CSS with 2.2.3 & panel 0.10.3: 
![image](https://user-images.githubusercontent.com/8346230/114152572-bb936600-991e-11eb-82a6-956631febc26.png)
vs. CSS not loaded with bokeh 2.3.0 & panel 0.11.1:
![image](https://user-images.githubusercontent.com/8346230/114152890-0f9e4a80-991f-11eb-9d15-f4d6799d889e.png)

**Comment by philippjfr** [comment]
Cannot reproduce this, could you indicate where you installed Panel 0.11.1 from?

**Comment by philippjfr** [comment]
Also can you look at your browser console and check whether there are any errors?

**Comment by JackDapid** [comment]
panel 0.11.1 installed via pip install - no error in JS console on chrome on two different machines, OSX and windows

**Comment by philippjfr** [comment]
Just tried it myself and could not reproduce. Can you try to shift-reload the page to ensure it's not something going on with your cache?

**Comment by JackDapid** [comment]
@philippjfr it made me curious why you can't reproduce ... I guess I found a reason that helps me :-)

For no good reason (just for historic reasons) I was always starting the app with: `bokeh serve ...` instead of `panel serve ...` - which never made so far a difference (as far as I know)

But now it makes a difference - not yet understood why.
I still see some formatting differences between 0.11.1. and 0.10.3 that might be css related - but will investigate further:

![Screenshot 2021-04-09 at 21 56 53](https://user-images.githubusercontent.com/8346230/114234112-9c2c2580-997e-11eb-9b30-04453496cee5.png)

**Comment by philippjfr** [comment]
>For no good reason (just for historic reasons) I was always starting the app with: bokeh serve ... instead of panel serve ... - which never made so far a difference (as far as I know)

Oh yeah, `panel serve` makes quite a few modifications to the server as it is launched. If you are serving any panel component you should definitely use it.

>I still see some formatting differences between 0.11.1. and 0.10.3 that might be css related - but will investigate further:

Suspect this is the layout regression in bokeh 2.3.0, which has since been fixed in 2.3.1.

**Comment by JackDapid** [comment]
Thanks, yes we historically started with a pure bokeh app, and only later added panel, and for the things I used never have seen any difference do kept using the bokeh serve - Thank you and closing this as rest part of the layout ssue I think is Bokeh 2.3.0 related
